app/locustio/jira/http_actions.py

```diff
@@ -184,9 +192,11 @@ def view_project_summary(locust):
 
     r = locust.client.get(f'/projects/{project_key}/summary', catch_response=True)
     content = r.content.decode('utf-8')
-    logger.info(f"{params.action_name}: View project {project_key}")
+    logger.locust_info(f"{params.action_name}: View project {project_key}")
 
     assert_string = f'["project-key"]="\\"{project_key}\\"'
+    if not (assert_string in content):
+        logger.error(f'{params.err_message} {project_key}')
```

Comment:
need to add content like:
logger.error(f'User {user[0]} authentication failed: {content}')